soem/ethercatmain.c

```diff
@@ -849,9 +849,39 @@ uint16 ecx_statecheck(ecx_contextt *context, uint16 slave, uint16 reqstate, int
    {
       if (slave < 1)
       {
+	 uint16 bitWiseState;
+	 uint16 slaveIndex;
```

Comment:
Remove the TABs, name the variables bitwisestate and slaveindex, no other variables in the function use camelcase